src/cclib/parser/gaussianparser.py

```diff
@@ -223,6 +223,109 @@ def extract(self, inputfile, line):
             natom = int(line.split()[1])
             self.set_attribute('natom', natom)
 
+        # Dipole moment
+        # e.g.
+        #  Dipole moment (field-independent basis, Debye):
+        #    X=              0.0000    Y=              0.0000    Z=              0.0930
+        if line[1:39] == "Dipole moment (field-independent basis":
```

Comment:
I think that if the ` (field-independent basis` part is removed from all the match checks, all the older regression tests that are failing will pass (rather than adding it to the hexadecapole line; then older logfiles won't have their `moments` parsed at all).